app/views/admin/neighbourhoods/show.html.erb

```diff
@@ -42,22 +48,34 @@
 <% if @neighbourhood.service_area_partners.present? %>
 <ul class="list-group">
   <% @neighbourhood.service_area_partners.order(:name).each do |partner| %>
-  <li class="list-group-item"><%= link_to partner.name, edit_admin_partner_path(partner) %></li>
+    <li class="list-group-item">
+      <% if current_user.partner_admin? -%>
+        <%= link_to partner.name, edit_admin_partner_path(partner) %>
+      <% else %>
+        <%= partner.name %>
+      <% end %>
+    </li>
   <% end %>
 </ul>
 <% end %>
 
-<hr>
-<h2>Sites</h2>
-<p>Found <%= @neighbourhood.sites.count %>.</p>
-<% if @neighbourhood.sites.present? %>
-<ul class="list-group">
-  <% @neighbourhood.sites.order(:name).each do |site| %>
-  <li class="list-group-item"><%= link_to site.name, edit_admin_site_path(site) %></li>
+
+<% if current_user.site_admin? %>
```

Comment:
neighbourhood admins and root users also need to be able to see all the info about what sites are attached to a neighbourhood. Neighbourhood admins should not then be able to edit a site but I have addressed this in my neighbourhood pr!

Reply:
i have updated this to use the same policy checks for [sites](https://github.com/geeksforsocialchange/PlaceCal/blob/febc6292df99ad11339dbd1bc9204b870effc316/app/policies/site_policy.rb#L24) and [neighbourhoods](https://github.com/geeksforsocialchange/PlaceCal/blob/febc6292df99ad11339dbd1bc9204b870effc316/app/policies/neighbourhood_policy.rb#L21).